packages/frontend/src/components/common/PivotTable/ValueCellMenu.tsx

```diff
@@ -138,6 +145,24 @@ const ValueCellMenu: FC<React.PropsWithChildren<ValueCellMenuProps>> = ({
         });
     };
 
+    const filterValue =
+        value.raw === undefined ||
+        (isDimension(item) && isDimensionValueInvalidDate(item, value))
+            ? null // Set as null if value is invalid date or undefined
```

Comment:
`isDimensionValueInvalidDate`
what's the reason for this?

Reply:
I copied this logic from `DashboardCellContextMenu` from regular tables, I guess that fixes and edge case with invalid dates.